docs/plugins/ingest-attachment.asciidoc

```diff
@@ -238,3 +238,58 @@ properties on this top level field will contain the value of the
 first attachment only. However, by specifying the
 `target_field` on to a value on `_ingest._value` it will correctly
 associate the properties with the correct attachment.
+
+[[ingest-attachment-raw-metadata]]
+==== Extracting all raw metadata from documents
+
+By default, the `ingest-attachment` plugin only extracts a subset of the most common metadata.
+
+If you want to get back all the raw metadata, you can set `properties` to `raw_metadata`.
+It will populate a subfield `raw_metadata` with all key/value pairs found as metadata in the document.
```

Comment:
should we mention that this can be a lot of fields and it might make more sense to select those one wants? Looks pretty verbose to me